docs/man-openmpi/man3/MPI_Accumulate.3.rst

```diff
@@ -33,8 +33,8 @@ C Syntax
    	MPI_Request *request)
 
 
-Fortran Syntax (see FORTRAN 77 NOTES)
```

Comment:
This is a note about your commit message for "doc: remove all fortran 77 notes"

I'm not sure that this is a correct statement: "Since 64 is the standard the typical int kind from MPI is 8." These values can vary from compiler to compiler, right?

Reply:
True, the exact kind number is compiler specific. I should have mentioned byte size 8.

Reply:
I still can't parse the commit message:

> Since 64 is the standard the typical int kind from MPI is the compiler kind corresponding to 8 bytes.

What does that mean?